action.yml

```diff
@@ -331,6 +379,10 @@ inputs:
     description: Shell command to prepend to the linter command
     required: false
     default: ""
+  erblint_auto_fix:
+    description: Whether this linter should try to fix code style issues automatically. If set to `true`, it will only work if "auto_fix" is set to `true` as well
+    required: false
+    default: "true"
```

Comment:
```suggestion
    default: "false"
```

Would it make sense to set the default to false if we know a linter doesn't support auto-fixing?

Reply:
That would fix these warnings with `auto_fix: true`:
![image](https://user-images.githubusercontent.com/14149056/163424545-438355f7-befe-45d3-baef-434f3ea65faf.png)

Reply:
Hi @ocean90, thanks for the feedback. You're right that does make more sense! We made these changes in our last commit